.github/pull_request_template.md

```diff
@@ -1,62 +1,61 @@
-<!--
-### How to use
-Leave the headings unless they don't apply to your PR, replace commented out text (surrounded with <!–– and ––>) with text describing your PR.
-NOTE: Please grant permission for repository maintainers to edit your PR.
-It is EXTREMELY common for PRs to be held up due to trivial changes being requested and the author being unavailable to make them.
+<!-- HOW TO USE: Under each "#### Heading" below, enter information relevant to your pull request.
+Leave the headings unless they don't apply to your PR, and remove the comment blocks (surrounded with <!–– and ––>) when you are done.
+
+NOTE: Please grant permission for repository maintainers to edit your PR.  It is EXTREMELY common for PRs to be held up due to trivial changes being requested and the author being unavailable to make them.
 -->
 
 #### Summary
-<!--
-A one-line description of your change that will be extracted and added to the [project changelog](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt).
 
-The format is (ignore the square brackets): ```SUMMARY: [Category] "[description]"```
+<!-- This section should consist of exactly one line, formatted like this:
+
+SUMMARY: [Category] "[Briefly describe the change]"
```

Comment:
```suggestion
SUMMARY: [Category] "[Briefly describe the change in these quotation marks]"
```
Not quite sure of the need for such a change, but it reminds of quotation marks.